enum/grundlageenergiemenge/grundlageenergiemenge.go

```diff
@@ -0,0 +1,28 @@
+package grundlageenergiemenge
+
+// GrundlageEnergiemenge gives hints about the foundation of an energy amount
+//go:generate stringer --type GrundlageEnergiemenge
+//go:generate jsonenums --type  GrundlageEnergiemenge
+type GrundlageEnergiemenge int
+
+const (
+	// ZAEHLERSTAND_ZUM_BEGIN_DER_ANGEGEBENEN_ENERGIEMENGE_VORHANDEN_UND_KOMMUNIZIERT
+	// corresponds to edi@energy qualifier Z36 "Zählerstand zum Beginn der angegebenen Energiemenge
+	// vorhanden und kommuniziert"
+	ZAEHLERSTAND_ZUM_BEGIN_DER_ANGEGEBENEN_ENERGIEMENGE_VORHANDEN_UND_KOMMUNIZIERT GrundlageEnergiemenge = iota + 1
```

Comment:
```suggestion
	ZAEHLERSTAND_ZUM_BEGINN_DER_ANGEGEBENEN_ENERGIEMENGE_VORHANDEN_UND_KOMMUNIZIERT GrundlageEnergiemenge = iota + 1
```
"zum Beginn" oder "zu beginn", in jedem fall mit doppel n ;)